### ISSUE TITLE
INFO  dm_ticket::dm > 提交订单失败, 原因:F-10012-01-16-001::亲，同一时间下单人数过多，建议您稍后再试, 此次抢购花费时间:932.014525ms

### ISSUE THREAD

**Issue by cya0401** [ClassmateLin/dm-ticket#49]
抢汪峰那种在网页内可看到场次的可以成功
但是五月天、黄绮珊这种网页内购买仅显示二维码不显示场次的成功率随机
![Screenshot from 2023-05-30 11-07-58](https://github.com/ClassmateLin/dm-ticket/assets/58257288/d6112301-4e3e-4c2c-8de0-846ded3fdbda)
![Screenshot from 2023-05-30 11-07-39](https://github.com/ClassmateLin/dm-ticket/assets/58257288/df68bf95-b2ec-4db5-9a5e-02d4ab504df9)
![Screenshot from 2023-05-30 11-05-52](https://github.com/ClassmateLin/dm-ticket/assets/58257288/97a9caed-9c8c-4501-ab62-28838a800e38)

**Comment by AnsonCong** [comment]
一样的错误 应该是去支付页面卡住了 
等待优化吧 最好要重复提交

**Comment by cya0401** [comment]
我先用汪峰、黄绮珊已经开售的做测试，两场均有余票，汪峰在网页内可见场次的可以成功，黄绮珊在网页内不可见场次的不成功。
然后切换到五月天试一把，倒数计时然后成功生成订单的时候着实让我激动了一把，可惜最后还是失败了。
不管怎样，感谢林同学，你就是我的超人

**Comment by cya0401** [comment]
我先用汪峰、黄绮珊已经开售的做测试，两场均有余票，汪峰在网页内可见场次的可以成功，黄绮珊在网页内不可见场次的不成功。
然后切换到五月天试一把，倒数计时然后成功生成订单的时候着实让我激动了一把，可惜最后还是失败了。
不管怎样，感谢林同学，你就是我的超人

**Comment by geraltgogo** [comment]
我和你一样, 也是同样的问题, 我抢五月天之前 我和我女朋友的孩子的名字我都想好了,  结果一愣神  发现同一时间抢报错之后就不会再次请求了。 反应过来的时候已经没票了。

**Comment by ClassmateLin** [comment]
> 一样的错误 应该是去支付页面卡住了 等待优化吧 最好要重复提交

不。没有进支付页面这回事。你手动抢也是报这个错误。

**Comment by cya0401** [comment]
> 一样的错误 应该是去支付页面卡住了 等待优化吧 最好要重复提交

我也觉得，估计是被盾了，然后幸运的是我朋友手动抢抢到了，哈哈哈

**Comment by geraltgogo** [comment]
@cya0401 内场票吗? 我想收了

**Comment by ClassmateLin** [comment]
> 我和你一样, 也是同样的问题, 我抢五月天之前 我和我女朋友的孩子的名字我都想好了, 结果一愣神 发现同一时间抢报错之后就不会再次请求了。 反应过来的时候已经没票了。

能看到这个，说明程序正常。提交订单失败, 原因:F-10012-01-16-001::亲，同一时间下单人数过多，建议您稍后再试, 此次抢购花费时间:932.014525ms

**Comment by AnsonCong** [comment]
>
是的 还是没有经验 如果手机端有这个问题 会立马重新提交 虽然也不一定有用 看看后期能否优化重复提交 和捡漏功能 总之感谢Lin佬

**Comment by cya0401** [comment]
> @cya0401 内场票吗? 我想收了

855的，准备自己看了，老哥加油

**Comment by AnsonCong** [comment]
> @cya0401 内场票吗? 我想收了

这次实名观看 咋收 要购票人的身份证入场

**Comment by geraltgogo** [comment]
> > @cya0401 内场票吗? 我想收了
> 
> 855的，准备自己看了，老哥加油
比杀了我还难受

**Comment by AnsonCong** [comment]
> 抢汪峰那种在网页内可看到场次的可以成功 但是五月天、黄绮珊这种网页内购买仅显示二维码不显示场次的成功率随机 ![Screenshot from 2023-05-30 11-07-58](https://user-images.githubusercontent.com/58257288/241838054-d6112301-4e3e-4c2c-8de0-846ded3fdbda.png) ![Screenshot from 2023-05-30 11-07-39](https://user-images.githubusercontent.com/58257288/241838078-df68bf95-b2ec-4db5-9a5e-02d4ab504df9.png) ![Screenshot from 2023-05-30 11-05-52](https://user-images.githubusercontent.com/58257288/241837908-97a9caed-9c8c-4501-ab62-28838a800e38.png)

黄绮珊好像要选座位 是不是不支持

**Comment by cya0401** [comment]
> > 抢汪峰那种在网页内可看到场次的可以成功 但是五月天、黄绮珊这种网页内购买仅显示二维码不显示场次的成功率随机 !
> 
> 黄绮珊好像要选座位 是不是不支持

不知道是不是这个原因，我唯一可以确定的这两的共性表现就是网页内只显示二维码，不像汪峰那种显示场次。
如果是选座位的话，在配置文件中没有添加相关语句，也有可能是失败的原因
